2021-06-26 23:29:16,583 - DEBUG - ServerEntity.LoadReporter - [ServerEntity.py:24]: __init__ create entity LoadReporter with id 60d747cce2557ddb151fb977 mem_id=1432164515552
2021-06-26 23:29:16,584 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d747cce2557ddb151fb977 
2021-06-26 23:29:16,596 - DEBUG - ServiceNode - [EtcdSupport.py:533]: we have create ServiceFinder
2021-06-26 23:29:16,604 - DEBUG - ServiceFinder - [EtcdSupport.py:352]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-06-26 23:29:16,612 - DEBUG - ServiceRegister - [EtcdSupport.py:155]: _do_regist: now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/$BATTLE_SRV_IP$|11001
2021-06-26 23:29:16,625 - CRITICAL - TcpServer - [TcpServer.py:208]: on_traceback, type:<class 'socket.gaierror'>, value:[Errno 11001] getaddrinfo failed, traceback:<class 'socket.gaierror'>
[Errno 11001] getaddrinfo failed
  File "C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpServer.py", line 188, in _start_server_task
    locals={'self': <TcpServer.TcpServer object at 0x0000014D72807190>, }
  File "C:\Program Files\Python38\lib\asyncio\streams.py", line 94, in start_server
    locals={'host': '$BATTLE_SRV_IP$', 'port': 11001, 'kwds': {}, 'client_connected_cb': <bound method TcpServer._handle_client_connected of <TcpServer.TcpServer object at 0x0000014D72807190>>, 'limit': 65536, 'loop': <ProactorEventLoop running=True closed=False debug=True>, 'factory': <function start_server.<locals>.factory at 0x0000014D73B94790>}
  File "C:\Program Files\Python38\lib\asyncio\base_events.py", line 1429, in create_server
    locals={'self': <ProactorEventLoop running=True closed=False debug=True>, 'protocol_factory': <function start_server.<locals>.factory at 0x0000014D73B94790>, 'host': '$BATTLE_SRV_IP$', 'sock': None, 'backlog': 100, 'ssl': None, 'reuse_address': False, 'reuse_port': None, 'ssl_handshake_timeout': None, 'start_serving': True, 'family': <AddressFamily.AF_UNSPEC: 0>, 'flags': <AddressInfo.AI_PASSIVE: 1>, 'port': 11001, 'sockets': [], 'hosts': ['$BATTLE_SRV_IP$'], 'fs': [<coroutine object BaseEventLoop._create_server_getaddrinfo at 0x0000014D73B98E40>]}
  File "C:\Program Files\Python38\lib\asyncio\base_events.py", line 1369, in _create_server_getaddrinfo
    locals={'self': <ProactorEventLoop running=True closed=False debug=True>, 'host': '$BATTLE_SRV_IP$', 'port': 11001, 'family': <AddressFamily.AF_UNSPEC: 0>, 'flags': <AddressInfo.AI_PASSIVE: 1>}
  File "C:\Program Files\Python38\lib\asyncio\base_events.py", line 1365, in _ensure_resolved
    locals={'self': <ProactorEventLoop running=True closed=False debug=True>, 'address': ('$BATTLE_SRV_IP$', 11001), 'family': <AddressFamily.AF_UNSPEC: 0>, 'type': <SocketKind.SOCK_STREAM: 1>, 'proto': 0, 'flags': <AddressInfo.AI_PASSIVE: 1>, 'loop': <ProactorEventLoop running=True closed=False debug=True>, 'host': '$BATTLE_SRV_IP$', 'port': 11001, 'info': None}
  File "C:\Program Files\Python38\lib\asyncio\base_events.py", line 825, in getaddrinfo
    locals={'self': <ProactorEventLoop running=True closed=False debug=True>, 'host': '$BATTLE_SRV_IP$', 'port': 11001, 'family': <AddressFamily.AF_UNSPEC: 0>, 'type': <SocketKind.SOCK_STREAM: 1>, 'proto': 0, 'flags': <AddressInfo.AI_PASSIVE: 1>, 'getaddr_func': <bound method BaseEventLoop._getaddrinfo_debug of <ProactorEventLoop running=True closed=False debug=True>>}
  File "C:\Program Files\Python38\lib\concurrent\futures\thread.py", line 57, in run
    locals={}
  File "C:\Program Files\Python38\lib\asyncio\base_events.py", line 808, in _getaddrinfo_debug
    locals={'self': <ProactorEventLoop running=True closed=False debug=True>, 'host': '$BATTLE_SRV_IP$', 'port': 11001, 'family': <AddressFamily.AF_UNSPEC: 0>, 'type': <SocketKind.SOCK_STREAM: 1>, 'proto': 0, 'flags': <AddressInfo.AI_PASSIVE: 1>, 'msg': '$BATTLE_SRV_IP$:11001, type=<SocketKind.SOCK_STREAM: 1>, flags=<AddressInfo.AI_PASSIVE: 1>', 't0': 623233.906}
  File "C:\Program Files\Python38\lib\socket.py", line 918, in getaddrinfo
    locals={'host': '$BATTLE_SRV_IP$', 'port': 11001, 'family': <AddressFamily.AF_UNSPEC: 0>, 'type': <SocketKind.SOCK_STREAM: 1>, 'proto': 0, 'flags': <AddressInfo.AI_PASSIVE: 1>, 'addrlist': []}
2021-06-26 23:29:16,641 - DEBUG - ServiceFinder - [EtcdSupport.py:323]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-06-26 23:29:16,644 - DEBUG - ServiceFinder - [EtcdSupport.py:277]: init service info from etcd success, will going to watch, at etcd index -> 186
2021-06-26 23:29:16,655 - DEBUG - ServiceRegister - [EtcdSupport.py:163]: regist service : battle_server success, {"action":"set","node":{"key":"/services/battle_server/$BATTLE_SRV_IP$|11001","value":"","expiration":"2021-06-26T15:32:16.6362784Z","ttl":180,"modifiedIndex":187,"createdIndex":187}}

2021-06-26 23:29:16,665 - DEBUG - ServiceFinder - [EtcdSupport.py:422]: recv watch set service node info--> battle_server: ('$BATTLE_SRV_IP$', 11001) - [187]
2021-06-26 23:30:18,268 - DEBUG - ServerEntity.LoadReporter - [ServerEntity.py:24]: __init__ create entity LoadReporter with id 60d7480a54b21e753e08ce8a mem_id=2678066724816
2021-06-26 23:30:18,269 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d7480a54b21e753e08ce8a 
2021-06-26 23:30:18,281 - DEBUG - ServiceNode - [EtcdSupport.py:533]: we have create ServiceFinder
2021-06-26 23:30:50,357 - DEBUG - ServerEntity.LoadReporter - [ServerEntity.py:24]: __init__ create entity LoadReporter with id 60d7482a4a716dec23152ca4 mem_id=2742760984480
2021-06-26 23:30:50,358 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d7482a4a716dec23152ca4 
2021-06-26 23:32:00,206 - DEBUG - ServerEntity.LoadReporter - [ServerEntity.py:24]: __init__ create entity LoadReporter with id 60d748708ac07d992e23fd34 mem_id=2298116063184
2021-06-26 23:32:00,207 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d748708ac07d992e23fd34 
2021-06-26 23:32:02,433 - DEBUG - ServiceNode - [EtcdSupport.py:533]: we have create ServiceFinder
2021-06-26 23:32:03,879 - DEBUG - ServiceFinder - [EtcdSupport.py:352]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-06-26 23:32:03,885 - DEBUG - ServiceRegister - [EtcdSupport.py:155]: _do_regist: now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:32:03,896 - DEBUG - TcpServer - [TcpServer.py:194]: Server on ('127.0.0.1', 11001)
2021-06-26 23:32:03,921 - DEBUG - ServiceFinder - [EtcdSupport.py:323]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-06-26 23:32:03,925 - DEBUG - ServiceFinder - [EtcdSupport.py:323]: add service node info--> battle_server: ('$BATTLE_SRV_IP$', 11001)
2021-06-26 23:32:03,929 - DEBUG - ServiceFinder - [EtcdSupport.py:277]: init service info from etcd success, will going to watch, at etcd index -> 189
2021-06-26 23:32:03,941 - DEBUG - ServiceRegister - [EtcdSupport.py:163]: regist service : battle_server success, {"action":"set","node":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:35:03.9158504Z","ttl":180,"modifiedIndex":190,"createdIndex":190}}

2021-06-26 23:32:03,953 - DEBUG - ServiceFinder - [EtcdSupport.py:422]: recv watch set service node info--> battle_server: ('127.0.0.1', 11001) - [190]
2021-06-26 23:32:17,046 - DEBUG - ServiceFinder - [EtcdSupport.py:433]: recv watch expire, info--> battle_server: ('$BATTLE_SRV_IP$', 11001) - [191]
2021-06-26 23:33:17,066 - DEBUG - ServiceFinder - [EtcdSupport.py:475]: etcd watch timeout, rest watch index 191 -> 191
2021-06-26 23:33:52,960 - DEBUG - ServiceRegister - [EtcdSupport.py:196]: outside, time.time()=1624721632.9572358, refresh service battle_server, now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:33:52,976 - DEBUG - ServiceRegister - [EtcdSupport.py:214]: time.time()=1624721632.9731805, refresh service success: battle_server: {"action":"set","node":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:36:52.9701895Z","ttl":180,"modifiedIndex":193,"createdIndex":193},"prevNode":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:35:03.9158504Z","ttl":71,"modifiedIndex":190,"createdIndex":190}}
, now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:34:20,080 - DEBUG - ServiceFinder - [EtcdSupport.py:475]: etcd watch timeout, rest watch index 191 -> 192
2021-06-26 23:36:58,215 - DEBUG - ServerEntity.LoadReporter - [ServerEntity.py:24]: __init__ create entity LoadReporter with id 60d7499a0a8d7b0297031f86 mem_id=2394059259856
2021-06-26 23:36:58,216 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d7499a0a8d7b0297031f86 
2021-06-26 23:36:58,229 - DEBUG - ServiceNode - [EtcdSupport.py:533]: we have create ServiceFinder
2021-06-26 23:36:58,243 - DEBUG - ServiceFinder - [EtcdSupport.py:352]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-06-26 23:36:58,249 - DEBUG - ServiceRegister - [EtcdSupport.py:155]: _do_regist: now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:36:58,260 - DEBUG - TcpServer - [TcpServer.py:194]: Server on ('127.0.0.1', 11001)
2021-06-26 23:36:58,280 - DEBUG - ServiceFinder - [EtcdSupport.py:323]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-06-26 23:36:58,283 - DEBUG - ServiceFinder - [EtcdSupport.py:277]: init service info from etcd success, will going to watch, at etcd index -> 196
2021-06-26 23:36:58,294 - DEBUG - ServiceRegister - [EtcdSupport.py:163]: regist service : battle_server success, {"action":"set","node":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:39:58.2756761Z","ttl":180,"modifiedIndex":197,"createdIndex":197}}

2021-06-26 23:36:58,305 - DEBUG - ServiceFinder - [EtcdSupport.py:422]: recv watch set service node info--> battle_server: ('127.0.0.1', 11001) - [197]
2021-06-26 23:38:06,329 - DEBUG - ServiceFinder - [EtcdSupport.py:475]: etcd watch timeout, rest watch index 197 -> 197
2021-06-26 23:38:39,323 - DEBUG - ServiceRegister - [EtcdSupport.py:196]: outside, time.time()=1624721919.3182304, refresh service battle_server, now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:38:39,342 - DEBUG - ServiceRegister - [EtcdSupport.py:214]: time.time()=1624721919.338184, refresh service success: battle_server: {"action":"set","node":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:41:39.3351949Z","ttl":180,"modifiedIndex":199,"createdIndex":199},"prevNode":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:39:58.2756761Z","ttl":79,"modifiedIndex":197,"createdIndex":197}}
, now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:39:13,346 - DEBUG - ServiceFinder - [EtcdSupport.py:475]: etcd watch timeout, rest watch index 197 -> 198
2021-06-26 23:40:16,360 - DEBUG - ServiceFinder - [EtcdSupport.py:475]: etcd watch timeout, rest watch index 198 -> 199
2021-06-26 23:40:24,367 - DEBUG - ServiceRegister - [EtcdSupport.py:196]: outside, time.time()=1624722024.3640738, refresh service battle_server, now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:40:24,383 - DEBUG - ServiceRegister - [EtcdSupport.py:214]: time.time()=1624722024.3800037, refresh service success: battle_server: {"action":"set","node":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:43:24.3779861Z","ttl":180,"modifiedIndex":201,"createdIndex":201},"prevNode":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:41:39.3351949Z","ttl":75,"modifiedIndex":199,"createdIndex":199}}
, now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:40:49,829 - DEBUG - ServerEntity.LoadReporter - [ServerEntity.py:24]: __init__ create entity LoadReporter with id 60d74a81695ea0e6c6b8e6aa mem_id=2848076218272
2021-06-26 23:40:49,830 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d74a81695ea0e6c6b8e6aa 
2021-06-26 23:40:49,842 - DEBUG - ServiceNode - [EtcdSupport.py:533]: we have create ServiceFinder
2021-06-26 23:40:49,853 - DEBUG - ServiceFinder - [EtcdSupport.py:352]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-06-26 23:40:49,860 - DEBUG - ServiceRegister - [EtcdSupport.py:155]: _do_regist: now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:40:49,870 - DEBUG - TcpServer - [TcpServer.py:194]: Server on ('127.0.0.1', 11001)
2021-06-26 23:40:49,893 - DEBUG - ServiceFinder - [EtcdSupport.py:323]: add service node info--> battle_server: ('127.0.0.1', 11001)
2021-06-26 23:40:49,896 - DEBUG - ServiceFinder - [EtcdSupport.py:323]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-06-26 23:40:49,899 - DEBUG - ServiceFinder - [EtcdSupport.py:277]: init service info from etcd success, will going to watch, at etcd index -> 202
2021-06-26 23:40:49,912 - DEBUG - ServiceRegister - [EtcdSupport.py:163]: regist service : battle_server success, {"action":"set","node":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:43:49.8876761Z","ttl":180,"modifiedIndex":203,"createdIndex":203},"prevNode":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:43:24.3779861Z","ttl":155,"modifiedIndex":201,"createdIndex":201}}

2021-06-26 23:40:49,925 - DEBUG - ServiceFinder - [EtcdSupport.py:422]: recv watch set service node info--> battle_server: ('127.0.0.1', 11001) - [203]
2021-06-26 23:40:49,928 - DEBUG - ServiceFinder - [EtcdSupport.py:299]: duplicate service info  -> battle_server:('127.0.0.1', 11001)
2021-06-26 23:41:29,089 - DEBUG - ServerEntity.LoadReporter - [ServerEntity.py:24]: __init__ create entity LoadReporter with id 60d74aa95cbda19299d83e6b mem_id=2002556501872
2021-06-26 23:41:29,089 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d74aa95cbda19299d83e6b 
2021-06-26 23:41:29,102 - DEBUG - ServiceNode - [EtcdSupport.py:533]: we have create ServiceFinder
2021-06-26 23:41:29,111 - DEBUG - ServiceFinder - [EtcdSupport.py:352]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-06-26 23:41:29,119 - DEBUG - ServiceRegister - [EtcdSupport.py:155]: _do_regist: now_url: http://127.0.0.1:2379/v2/keys/services/battle_server/127.0.0.1|11001
2021-06-26 23:41:29,130 - DEBUG - TcpServer - [TcpServer.py:194]: Server on ('127.0.0.1', 11001)
2021-06-26 23:41:29,154 - DEBUG - ServiceFinder - [EtcdSupport.py:323]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-06-26 23:41:29,157 - DEBUG - ServiceFinder - [EtcdSupport.py:277]: init service info from etcd success, will going to watch, at etcd index -> 4
2021-06-26 23:41:29,171 - DEBUG - ServiceRegister - [EtcdSupport.py:163]: regist service : battle_server success, {"action":"set","node":{"key":"/services/battle_server/127.0.0.1|11001","value":"","expiration":"2021-06-26T15:44:29.1489245Z","ttl":180,"modifiedIndex":5,"createdIndex":5}}

2021-06-26 23:41:29,183 - DEBUG - ServiceFinder - [EtcdSupport.py:422]: recv watch set service node info--> battle_server: ('127.0.0.1', 11001) - [5]
2021-06-26 23:41:44,849 - DEBUG - TcpServer - [TcpServer.py:180]: ('127.0.0.1', 13526) is connected !!!!
2021-06-26 23:41:44,885 - DEBUG - ServerEntity.Puppet - [ServerEntity.py:24]: __init__ create entity Puppet with id 60d74ab85cbda19299d83e6c mem_id=2002557288704
2021-06-26 23:41:44,889 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60d74ab85cbda19299d83e6c 
